src/js/components/MarathonTaskDetailsList.js

```diff
@@ -58,6 +58,14 @@ class MarathonTaskDetailsList extends React.Component {
     return task.status;
   }
 
+  getTimeField(time) {
+    return (
+      <time dateTime={time} title={time}>
+        {new Date(time).toLocaleString()}
```

Comment:
This is directly taken from Marathon UI and I remember it was a requirement from a user to do it with locale strings.